### PR TITLE
Add `_replace_fill_values` to Internal API

### PR DESCRIPTION
**Pull request by philipc2**
Hotfix to recent merge of #241 

# Overview
* Added `_replace_fill_values` to Internal API 
* Added "optional" tag to `new_dtype` arg

### CODE REVIEW

**Comment by rajeeja** [comment]
can you add if it is a required or optional arg e.g.:

new_dtype (np.dtype)  

to

`new_dtype (np.dtype, optional) `

File: uxarray/helpers.py [hunk 1]
```diff
@@ -574,7 +574,7 @@ def _replace_fill_values(grid_var, original_fill, new_fill, new_dtype=None):
         original fill value used in (``grid_var``)
     new_fill : constant
         new fill value to be used in (``grid_var``)
-    new_dtype : np.dtype
+    new_dtype : (np.dtype, optional)
```
Comment (anissa111) [inline uxarray/helpers.py:577]:
```suggestion
    new_dtype : np.dtype, optional
```

**Review by anissa111 — COMMENTED** [review]
image

The docs here still look a little funny. I would suggest replacing `constant` with `float` or `int` if appropriate and replacing `np.ndarray` with `numpy.ndarray` for proper linkage. I've also provided a suggestion below that will fix the double parentheses on the `new_dtype` parameter entry.

**Review by hCraker — APPROVED** [review]
Looks good once you address Anissa's suggestions

**Review by rajeeja — APPROVED** [review]

**Pull request merged**